src/runtime/manifest-primitive-parser/smoke/toy.html

```diff
@@ -0,0 +1,161 @@
+<style>
+  body {
+    height: 100vh;
+    font-family: sans-serif;
+    display: flex;
+    flex-direction: column;
+  }
+  body, pre {
+    margin: 0;
+    box-sizing: border-box;
+  }
+  pre {
+    color: navy;
+    background-color: #FAFAFA;
+  }
+  [columns] {
+    display: flex;
+    align-items: stretch;
+    height: 80vh;
+  }
+  [column] {
+    flex: 1;
+    border: 1px dotted silver;
+    display: flex;
+    flex-direction: column;
+    overflow: auto;
+  }
+  [flex] {
+    flex: 1;
+  }
+  #logElt {
+    padding: 12px;
+  }
+  textarea {
+    flex: 1;
+    padding: 4px;
+    border: none;
+    outline: none;
+  }
+</style>
+
+<div columns>
+  <div column>
+    <textarea id="inputElt" spellcheck="false"></textarea>
+  </div>
+  <div column id="treeElt">
+    <pre id="outputElt"></pre>
+  </div>
+</div>
+<pre flex id="logElt"></pre>
+
+<script type="module">
+  import {Loader} from '../../../../build/platform/loader.js'
+  import {ManifestPrimitiveParser} from '../../../../build/runtime/manifest-primitive-parser/manifest-primitive-parser.js';
+
+  const dom = (tag, props, parent) => parent.appendChild(Object.assign(document.createElement(tag), props));
+  const {inputElt, logElt, outputElt} = window;
+
+  const loader = new Loader();
+
+  inputElt.oninput = async () => {
+    try {
+      logElt.innerText = '';
+      const content = inputElt.value;
+      const opts = {
+        loader,
+        filename: `http://localhost/projects/arcs/arcs/particles/Bespoke/Bespoke.arcs`
```

Comment:
Hard-coded path specific to your usage? I presume you're serving from root and `projects/arcs/arcs` is the repo location.

Also, there is no `Bespoke/Bespoke.arcs`.

Reply:
I copied `toy.html` in at the last minute, sorry I failed to update that path.

There is no "Bespoke/Bespoke.arcs" because it's a faux name for the inline content (it's not actually read from disk). It must have a filename for relative import statements to resolve against. 

The filename's path is wrong on your machine(s) which is why the sample is not parsing for you.

Reply:
I did try changing this to match my serving location and repo name, but it  still failed to load with the import statement left in.

Reply:
I'd have to see your changes to explain that. But in any case, the latest version should not have this problem.

Reply:
Yep, works now.